src/sentry/testutils/cases.py

```diff
@@ -2358,7 +2358,7 @@ def setUp(self):
         # Token based auth
         sentry_app = self.create_sentry_app(
             organization=self.organization,
-            scopes=["project:write"],
+            scopes=["project:read", "project:write"],
```

Comment:
the `MonitorSerializer` expects `project:read` but we document `read/write/admin`
https://docs.sentry.io/api/crons/update-a-checkin/

Any way to fix the validator?

Reply:
where does it require this?

Reply:
https://github.com/getsentry/sentry/blob/c1f8632bd6972cbd0121f77ed980572b4c0bff88/src/sentry/monitors/validators.py#L152

Reply:
https://github.com/getsentry/sentry/blob/37c3ee8184b1d19efc84990e4b96d0ef91bcc3a6/src/sentry/api/serializers/rest_framework/project.py#L11

Reply:
Hmm, what do you mean fix then, do you mean it should check multiple scopes?

Reply:
yeah otherwise our validator is being overly harsh and not to documented spec

Reply:
hmm, this is an existing bug though right?

Reply:
looks like it...i can cut a side bugfix I guess

Reply:
https://docs.sentry.io/api/permissions/
re-reading the docs this is not a bug but intended. so we can either change the spec or just scrub the data so it doesn't even get passed to the validator